libraries/botbuilder-repo-utils/src/file.ts

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import fs from 'fs';
+import util from 'util';
```

Comment:
Do you prefer to use this import syntax instead of `const { promisify } from 'util';` for core Node modules? We're not uniform on import syntax so this is an opportunity to set ground rules for future cleanup 😄 

Have you tried using `fs/promises`? ([Node API Docs example](https://nodejs.org/docs/latest-v12.x/api/fs.html#fs_promise_example))

Reply:
That's handy! In general, I think we should always favor `import` in any Typescript code.

Reply:
Sorry! I meant `import`, not `const`. It was more the named imports of the specific classes/functions versus importing the entire module. I fixed my earlier comment.

Reply:
Everything has counterexamples, so I'll try to back up my views.

In general, I prefer the simplicity of importing a full module by name. Conflicts are less likely, and it doesn't pollute the module scope.

That said, if module or bundle size is a concern (i.e., code, especially bundled code we ship to users), then it's more important, in my opinion, to carefully control the bundle size. Lodash isolated imports are an example of this.

Reply:
One last thing worth mentioning is method binding: if the default export is an instance of a class, it can be dangerous to selectively import methods as they won't have the correct ’this’ value when invoked. It can be non-trivial to know when this is the case due to the flexible nature of Node packages, so definitely worth keeping in mind.

Reply:
One final last thing: if the documentation makes it obvious the individual imports are preferred or more natural, like importing class constructors, then, by all means, things should be specifically imported.

Reply:
Sound reasoning, thanks for the in-depth answers! FYI @mdrichardson @Zerryth @EricDahlvang